workflow/controller/operator_test.go

```diff
@@ -10870,6 +10870,7 @@ func TestContainerSetDeleteContainerWhenPodDeleted(t *testing.T) {
 		}
 	}
 
+	// TODO: Enable use local variables in the test to avoid long wait. See https://github.com/argoproj/argo-workflows/pull/12756#discussion_r1530245007
```

Comment:
```suggestion
	// TODO: Refactor to use local-scoped env vars in test to avoid long wait. See https://github.com/argoproj/argo-workflows/pull/12756#discussion_r1530245007
```

Small clarification